application/email/v0/config/tasks.json

```diff
@@ -0,0 +1,471 @@
+{
+  "$defs": {
+    "search-subject-text": {
+      "description": "Search for emails with a specific subject",
+      "instillAcceptFormats": [
+        "string"
+      ],
+      "instillUpstreamTypes": [
+        "value",
+        "reference",
+        "template"
+      ],
+      "instillUIOrder": 4,
+      "title": "Search Subject",
+      "type": "string"
+    },
+    "search-from-email": {
+      "description": "Search for emails from a specific email address",
+      "instillAcceptFormats": [
+        "string"
+      ],
+      "instillUpstreamTypes": [
+        "value",
+        "reference",
+        "template"
+      ],
+      "instillUIOrder": 2,
+      "title": "Search From",
+      "type": "string"
+    },
+    "search-to-email": {
+      "description": "Search for emails to a specific email address",
+      "instillAcceptFormats": [
+        "string"
+      ],
+      "instillUpstreamTypes": [
+        "value",
+        "reference",
+        "template"
+      ],
+      "instillUIOrder": 3,
+      "title": "Search To",
+      "type": "string"
+    },
+    "server-address": {
```

Comment:
I'm thinking we should put the `server-address` and `server-port` into `setup.json`. They should be in the same place as the user email and password.